react-carousel-component/my-app/src/Carousel.js

```diff
@@ -0,0 +1,61 @@
+import './Carousel.css';
+import { useState } from 'react';
+import { useEffect } from 'react';
+
+export default function Carousel({ images }) {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  useEffect(() => {
+    let carousel = setInterval(handleRight, 1500);
+    return () => clearInterval(carousel);
+  });
```

Comment:
How often does the effect run without a dependency array? How often do you _want_ it to run?

Remember that every time the effect runs you will call the cleanup function, which will cancel your interval.